docs/CHANGES.md

```diff
@@ -4,6 +4,7 @@
 
 - #585 Duplicate contact created due to JID case sensivity [saganshul]
 - #628 Fixes the bug in displaying chat status during private chat. [saganshul]
+- #797 Time format made configurable, with a choice between 12hr and 24hr format. [smitbose]
```

Comment:
yeah @jcbrand I think this 12hr/24hr thing has got embedded in my head! :P I keep repeating the same damn thing every single time! :D